test/uk/gov/hmrc/helptosaveproxy/controllers/HelpToSaveControllerSpec.scala

```diff
@@ -36,12 +37,15 @@ import uk.gov.hmrc.http.{HeaderCarrier, HttpResponse}
 
 import scala.concurrent.{ExecutionContext, Future}
 
-class HelpToSaveControllerSpec extends TestSupport {
+class HelpToSaveControllerSpec extends AuthSupport {
 
   val mockNSIConnector = mock[NSIConnector]
   val mockJsonSchema = mock[JSONSchemaValidationService]
 
-  val controller = new HelpToSaveController(mockNSIConnector, mockJsonSchema)
+  val controller = new HelpToSaveController(mockNSIConnector, mockJsonSchema, mockAuthConnector)
+
+  val ggCreds = Credentials("id", "GovernmentGateway")
+  val ggRetrievals: Option[String] ~ Option[Credentials] = new ~(Some(nino), Some(ggCreds))
```

Comment:
i think we don't need to retrieve nino

Reply:
Ok I have changed it to a None and unit tests still pass

Reply:
you dont even have to pass `None`